bpf/bpf_lxc.c

```diff
@@ -1293,6 +1321,11 @@ int tail_ipv4_policy(struct __ctx_buff *ctx)
 
 	/* Store meta: essential for proxy ingress, see bpf_host.c */
 	ctx_store_meta(ctx, CB_PROXY_MAGIC, ctx->mark);
+
+	if (ret != CTX_ACT_TX && !encode_custom_prog_meta(ctx, ret, src_label))
```

Comment:
Maybe a comment would be useful here to explain the first part of the condition is for the proxy case?

Reply:
Agreed, I was careful to document in the commit log, but forgot the comment. Will fix.